x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/middleware.ts

```diff
@@ -80,16 +80,18 @@ export const hostMiddlewareFactory: ImmutableMiddlewareFactory<HostState> = core
                     version: '1.0.0',
                     status: HostPolicyResponseActionStatus.success,
                     id: '17d4b81d-9940-4b64-9de5-3e03ef1fb5cf',
-                    actions: {
-                      download_model: {
+                    actions: [
```

Comment:
you probably added this only because @parkiino has not yet merged her change that integrates with API, correct (and to suppress ts errors)?

Reply:
Yes just to get the test to pass.